### PR TITLE
Create Why-to-contribute.md

### PR DESCRIPTION
**Pull request by psmulovics**

### CODE REVIEW

**Comment by psmulovics** [comment]
Sure, this is just the summary of my OSFF presentation from 2021. Should I add a frontmatter draft on it?

**Comment by psmulovics** [comment]
* Move it to Level 4, full contribution
* Add frontmatter

**Review by robmoffat — APPROVED** [review]

File: docs/bok/Activities/Level-3/Why-to-contribute.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# "Why Going Open Source?"
+
+Many reasonings can be defined why would someone start contributing to open source. When it comes to a financial company, it's not different, but might be a different reasoning behind. So, why would do a financial company go and contribute to open source:
+
+ * There is a marketing value to talk about by the corporate communications teams
+ * There is a hiring value to talk about by the human resources teams
+ * There is the value that the contribution would be used by others
```
Comment (tsteenbe) [inline docs/bok/Activities/Level-3/Why-to-contribute.md:7]: Would add "Influence communities of strategic importance to the organization"

**Comment by psmulovics** [comment]
Link to 'making the case' for talent retention

**Comment by robmoffat** [comment]
@psmulovics I've made a few grammatical/stylistic changes to (hopefully) make your doc easier to parse.  

Do you want to add those things from the SIG meeting or merge for now?

**Comment by psmulovics** [comment]
> Do you want to add those things from the SIG meeting or merge for now?

I feel like those items are now covered adequately, so I think we can merge safely.

**Comment by robmoffat** [comment]
well done!

**Pull request merged**